cmd/tls_integration_test.go

```diff
@@ -0,0 +1,71 @@
+//go:build !windows
+// +build !windows
```

Comment:
```suggestion
```
Did you find issues on Windows?

Reply:
> Actually yes if you look into the test failure you will see how it fails with
> 
> > crypto/x509: system root pool is not available on Windows
> 
> I don't know though why it only happens for the 1.17.x 🤷
> 
> edit: seems to have been fixed with [golang/go@3544082](https://github.com/golang/go/commit/3544082f75fd3d2df7af237ed9aef3ddd499ab9c) ... so this might be blocked on us dropping 1.17.x support - very likely for the next release when we move to 1.19.x as the primary version we release with

I thought some windows tests were running against 1.17.x golang version, my bad !